azure-pipelines-with-e2etests_teamsV1.yml

```diff
@@ -0,0 +1,80 @@
+# This pipeline checks-out current client sdk repo and stable app hosting sdk repo (as specified in AppHostingSdkGitRef).
```

Comment:
Now that I'm thinking about it, I don't think we want to run these tests as part of TeamsJs v2 validation. No changes we make here should affect this. We definitely want to run it as part of validating the host part in the other repo in ADO.

Reply:
On the other hand, we might want to run it to make sure the test app is not changed in a way that would make the tests fail. What do you think Ella?

Reply:
Discussed offline. We will keep the pipeline here.